docs/src/whatsnew/latest.rst

```diff
@@ -203,6 +203,9 @@ This document explains the changes made to Iris for this release
 #. `@bjlittle`_ migrated readthedocs to use mambaforge for `faster documentation building`_.
    (:pull:`4476`)
 
+#. `@wjbenfold`_ contributed `@alastair-gemmell`_'s step-by-step guide to
+   contributing to the docs to the docs. (:pull:`4461`)
```

Comment:
@wjbenfold Sorry... I'll stop after this... but would you consider adding a link to the doc changes here in the `whatsnew` entry... then it's a simple one-click for readers to follow-thru and easily see this change